package.json

```diff
@@ -1,68 +1,65 @@
-    "passport": "~0.1.17",
-    "passport-facebook": "~2.0.0",
-    "passport-github": "~0.1.5",
-    "passport-google-oauth": "~0.1.5",
-    "passport-local": "~0.1.6",
-    "passport-twitter": "~1.0.2",
-    "sinon": "~1.7.3",
-    "socket.io": "~0.9.16",
-    "socket.io-client": "~0.9.16",
-    "underscore": "~1.5.2",
-    "view-helpers": "~0.1.3"
-  },
-  "devDependencies": {
-    "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-plugin-import": "^2.8.0",
-    "grunt": "~0.4.1",
-    "grunt-concurrent": "~0.4.0",
-    "grunt-contrib-jshint": "~0.6.4",
-    "grunt-contrib-sass": "~0.5.0",
-    "grunt-contrib-watch": "~0.5.3",
-    "grunt-mocha-test": "~0.7.0",
-    "grunt-nodemon": "~0.1.1",
-    "jasmine": "^2.8.0",
-    "jasmine-core": "^2.8.0",
-    "karma": "^1.7.1",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-jasmine": "^1.1.0",
-    "should": "~1.3.0",
-    "supertest": "~0.8.0"
-  }
+    "name": "cardsforhumanity",
+    "description": "A Game For Horrible People Desperately Trying To Do Good",
+    "version": "1.0.0",
+    "private": false,
+    "author": "cardsforhumanity",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/cardsforhumanity/cardsforhumanity.git"
+    },
+    "engines": {
+        "node": "0.10.x",
+        "npm": "1.2.x"
+    },
+    "scripts": {
+        "bower": "node node_modules/gulp/bin/gulp install",
+        "start": "node node_modules/gulp/bin/gulp",
+        "test": "node node_modules/gulp/bin/gulp test",
+        "test:run": "karma start karma.config.js"
+    },
+    "dependencies": {
+        "async": "~0.2.9",
+        "bcryptjs": "~1.0.4",
+        "bower": "~1.3.8",
+        "connect-flash": "~0.1.1",
+        "connect-mongo": "~0.3.3",
+        "console-stamp": "~0.1.1",
+        "dotenv": "^4.0.0",
+        "express": "~3.4.0",
+        "forever": "latest",
+        "gulp-cli": "^1.4.0",
+        "jade": "~0.35.0",
+        "mean-logger": "~0.0.1",
+        "mongoose": "~3.6.20",
+        "passport": "~0.1.17",
+        "passport-facebook": "~2.0.0",
+        "passport-github": "~0.1.5",
+        "passport-google-oauth": "~0.1.5",
+        "passport-local": "~0.1.6",
+        "passport-twitter": "~1.0.2",
+        "sinon": "~1.7.3",
+        "socket.io": "~0.9.16",
+        "socket.io-client": "~0.9.16",
+        "underscore": "~1.5.2",
+        "view-helpers": "~0.1.3"
+    },
+    "devDependencies": {
+        "browser-sync": "^2.18.13",
+        "eslint-config-airbnb-base": "^12.1.0",
+        "eslint-plugin-import": "^2.8.0",
+        "gulp": "^3.9.1",
+        "gulp-bower": "0.0.13",
+        "gulp-eslint": "^4.0.0",
+        "gulp-mocha": "^4.3.1",
+        "gulp-nodemon": "^2.2.1",
+        "gulp-sass": "^3.1.0",
+        "should": "~1.3.0",
+        "supertest": "~0.8.0",
+        "jasmine": "^2.8.0",
+        "jasmine-core": "^2.8.0",
+        "karma": "^1.7.1",
+        "karma-chrome-launcher": "^2.2.0",
+        "karma-jasmine": "^1.1.0"
+    }
 }
```

Comment:
We need to update most of these packages to the latest version. For example, jade is now pug. Running `npm install` with the current package.json content shows so many deprecated packages, some of which are no longer supported. This is out of scope for this task which is `Convert from Grunt to Gulp taskrunner`. The package updates can be done as another task

Reply:
Ok